embiggen/sequences/abstract_node2vec_sequence.py

```diff
@@ -69,6 +65,9 @@ def __init__(
             Weight on the probability of visiting a neighbor edge of a
             different type than the previous edge. This only applies to
             multigraphs, otherwise it has no impact.
+        max_neighbours: int = None,
+            Number of maximum neighbours to consider when using approximated walks.
```

Comment:
Possibly could add "(useful for graphs containing nodes with extremely high degrees)" - readers might not know what 'approximated walks" are

Reply:
Sure! On it!